packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offers/styles.less

```diff
@@ -0,0 +1,24 @@
+dedicated-cloud-datacenter-backup-offers {
+  .oui-select-picker__body {
+    flex: 1;
+
+    .oui-select-picker__label {
+      padding-left: 0;
+      text-transform: capitalize;
+
+      &::before,
+      &::after {
+        display: none;
+      }
+    }
+
+    .oui-select-picker__section {
+      border-top: none;
+    }
+  }
```

Comment:
Why is some custom style is needed ? Maybe we could keep current styles and that style override comes as a theme/larger revamp

Reply:
But that wont conform to the approved design. I have to see if it is okay for the product release.

Reply:
Are you sure the lower one:
- will always match your capacity (for example `renew`), even in the future ?
- will always match the lower price, even in the future ?

Please use the `ovhManagerOrderCatalogPrice` component instead, it already handles all those cases :)

Reply:
The price we are displaying is not `prorata` and we are saying `"Price starts at ---/VM"`. So it should stay constant. And only one `price/month` is obtained from API. So this should suffice